backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -163,10 +176,11 @@ func Get(reg registry.Registry) *Handler {
 		round:            0,
 		gameState:        WaitForStart,
 		selectTopicTimer: 10 * time.Second,
-		playTimeTimer:    60 * time.Second,
-		playTimeChan:     make(chan PlayTimeReceive),
+		playTimeTimer:    5 * time.Second,
```

Comment:
This playtime is just to make things easier to test rn, I can change it back to something legit later